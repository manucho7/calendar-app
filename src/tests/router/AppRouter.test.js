import React from 'react';
import{ mount } from 'enzyme';
import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import thunk from 'redux-thunk';
import { AppRouter } from '../../router/AppRouter';

import '@testing-library/jest-dom';
import '@testing-library/dom';


const middlewares = [thunk];
const mockStore = configureStore( middlewares );

// store.dispatch = jest.fn();

describe('Pruebas en AppRouter', () => {
    
    test('debe de mostrar el espere...', () => {
        
        const initState = {
            auth: {
                checking: true
            }
        };

        const store = mockStore( initState );

        const wrapper = mount(
            <Provider store={store}>
                <AppRouter />
            </Provider>
        );

        expect( wrapper ).toMatchSnapshot();

    });
    

})
