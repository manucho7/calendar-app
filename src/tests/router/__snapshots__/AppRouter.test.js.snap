// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en AppRouter debe de mostrar el espere... 1`] = `
<h5>
  Espere...
</h5>
`;
